src/report/ReportManager.ts

```diff
@@ -572,11 +594,14 @@ class Help implements IUIAction {
     public async title(_manager: ReportManager, _report: IReport): Promise<string> {
         return "Help";
     }
+    public async help(_manager: ReportManager, _report: IReport): Promise<string> {
+        return "This help";
+    }
     public async execute(manager: ReportManager, report: IReport): Promise<string | undefined> {
         // Produce a html list of actions, in the order specified by ACTION_LIST.
         let list: string[] = [];
         for (let action of ACTION_LIST) {
-            list.push(`<li>${action.emoji} ${await action.title(manager, report)}</li>`);
+            list.push(`<li>${action.emoji} ${await action.help(manager, report)}</li>`);
```

Comment:
This is alright but at the moment it shows help for escalation even though I'm not sure that's implemented yet. it will also show help for actions even when you can't do them because there isn't the right permissions available and I don't know if it's made clear why you can't use them (e.g. if it was reported from an unprotected room).

Reply:
Updated. Is this better?

Reply:
Yeah that does the trick, but it is now saying 

>Some actions were disabled, usually because the event took place in a room where Mjölnir is not moderator.

Because the escalation command doesn't work anywhere yet

Reply:
Good catch. Rewritten again.